doc/source/abfall_neunkirchen_siegerland_de.md

```diff
@@ -0,0 +1,38 @@
+# Neunkirchen Siegerland
+
+Support for schedules provided by [Gemeinde Neunkirchen Siegerland](https://www.neunkirchen-siegerland.de/), Germany.
+
+## Configuration via configuration.yaml
+
+```yaml
+waste_collection_schedule:
+  sources:
+    - name: abfall_neunkirchen_siegerland_de
+      args:
+        strasse: STRASSE
+```
+
+### Configuration Variables
+
+**searchterm**<br>
```

Comment:
`strasse` vs `searchterm` - which one is correct?

Please don't use `searchterm` as name. I would prefer `strasse`.

Reply:
changed searchterm to strasse